glide.lock

```diff
@@ -1,33 +1,36 @@
-hash: 111bdf9797d23d4b9f8833a9780e382dd53153af70ad286f99815cea4c2fdf4b
-updated: 2016-08-05T18:02:17.171302481-04:00
+hash: 96546aa202f8964e8cc0c8acea5452cc9c6d79259ea4f9aaff03ffbf13c918a4
```

Comment:
Can we do the glide change in a separate PR -- since it upgrades a lot of other deps too. Let's make this change just affect the thrift compiler